test/e2e-olm/utils.go

```diff
@@ -57,7 +64,7 @@ func elasticsearchNameFor(uuid string) string {
 	return fmt.Sprintf("%s-%s", elasticsearchCRName, uuid)
 }
 
-func createElasticsearchCR(t *testing.T, f *test.Framework, ctx *test.Context, esUUID, dataUUID string, replicas int) (*loggingv1.Elasticsearch, error) {
+func createElasticsearchCR(t *testing.T, f *test.Framework, ctx *test.Context, esUUID, dataUUID string, replicas int, storageClassType StorageClassType, storageClassName *string) (*loggingv1.Elasticsearch, error) {
```

Comment:
i think based on my prior comments we can back this out

Reply:
Based on the fact that the E2E tests for ephemeral and nonDefaultStorageClass are removed now, the above change is also obsolete.